static/js/enketo/widgets/notewidget.js

```diff
@@ -63,6 +69,24 @@ define( function( require, exports, module ) {
         /* jshint unused:false */
     };
 
+    function applyLiveLinkHtml( $el ) {
+        var html = $el.html();
+
+        html = html.replace( /\[([^\]]*)\]\(([^)]*<[^>]*\>[^)]*)\)/gm,
+                '<a class="live-link" href="#" target="_blank">$1<span class="href">$2</span></a>' );
```

Comment:
Then the links wouldn't have dynamic hrefs - that's the bug being fixed in https://github.com/medic/medic-webapp/issues/3349